README.md

```diff
@@ -38,6 +38,7 @@ The NAB scores are normalized such that the maximum possible is 100.0 (i.e. the
 | [Twitter ADVec v1.0.0](https://github.com/twitter/AnomalyDetection)| 47.1             | 33.6          | 53.5          |
 | [Etsy Skyline](https://github.com/etsy/skyline) | 35.7             | 27.1          | 44.5          |
 | Bayesian Changepoint**          | 17.7              | 3.2           | 32.2           |
+|  EXPoSE   | 16.4     | 3.2  | 26.9     |
```

Comment:
Should it be linked to this one or the original expose paper https://arxiv.org/pdf/1601.06602v3.pdf? The implementation is for the original paper w/o stochastic optimization.